__tests__/fixtures/fakeContentCreator.js

```diff
@@ -0,0 +1,11 @@
+const { encrypt } = require('../../helpers/password');
+const mongoose = require('mongoose');
+
+module.exports = function makeFakeCreator() {
+  return {
+    email: "test1@mail.dk",
+    password: encrypt('ABC123456!'),
+    name: "test1 test1",
```

Comment:
Dette er ændret til first name og last name i mobile for users.
Ved ikke hvordan det er for content creators, men det kunne være nice hvis i tilføjede denne model til models.md i ressources og til miro, og ændrede den, hvis den skal ændres. :-)